qframe_test.go

```diff
@@ -1373,6 +1373,41 @@ mon
 	})
 }
 
+func TestQFrame_ReadCSVMissingColumnName(t *testing.T) {
+	input := `,COL2
+a,1.5`
+	expectedIn := `COL,COL2
+a,1.5`
+
+	out := qframe.ReadCSV(strings.NewReader(input), csv.MissingColumnNameAlias("COL"))
+	expected := qframe.ReadCSV(strings.NewReader(expectedIn))
+	assertEquals(t, expected, out)
+}
+
+func TestQFrame_ReadCSVDuplicateColumnName(t *testing.T) {
```

Comment:
This test does not assert anything meaningful ATM, there is an error in the header (a missing comma between col two and three) that cause the read method to fail. The resulting frame has `Err != nil`. Please fix this and also perhaps add a check that there is no error when reading the frame (`assertNotErr(t, out.Err)`) to be on the safe side.